<launch>

  <arg name="stdout" default="screen" />
  <arg name="camera_name" default="/zed2i/zed_node/" />
  <arg name="image_topic" default="image_rect_color" />
  <arg name="frame_name" default="zed2i_left_camera_frame"/>
  <arg name="sensor_id" default="0" />

  <!-- Our node -->
  <node pkg="velo2cam_stereoHough" type="my_depth" name="my_depth_ORB" output="screen">
    <remap from="image_left" to="$(arg camera_name)/left/$(arg image_topic)" />
    <remap from="image_right" to="$(arg camera_name)/right/$(arg image_topic)" />
  </node>

</launch>
